react/Labs/ExperimentalDialog/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+
+import Dialog from '@material-ui/core/Dialog'
+import DialogTitle from '@material-ui/core/DialogTitle'
+import DialogActions from '@material-ui/core/DialogActions'
+import withMobileDialog from '@material-ui/core/withMobileDialog'
+import PropTypes from 'prop-types'
+import AppTitle from '../../AppTitle'
+
+export const ExperimentalDialog = props => {
+  const { onClose, fullScreen, children, open, scroll } = props
+  return (
+    <Dialog
+      open={open}
+      onClose={onClose}
+      fullScreen={fullScreen}
+      aria-labelledby="dialog-title"
+      scroll={scroll}
```

Comment:
Can we spread the `props` here? That way if we want to add other props supported by Dialog, we don't need to add them here.